templates/applicantForm.phtml

```diff
@@ -53,9 +53,8 @@
                 </div>
 
                 <div id="application" class="collapse">
-                    <label for="cohorts">When would you like to join us?</label>
-                    <select id="cohorts" class="submitApplicant form-control" name="cohortId"  data-selected="<?php echo $applicant->getCohortId();?>" required>
-                    </select>
+                    <label>When would you like to join us?</label>
+                    <div id="cohorts" data-dates="<?php echo $applicant->getCohortIds();?>"></div>
```

Comment:
can we add some JS validation to stop someone from submitting the form if none of the checkboxes are checked. fr both the add and edit page